GlotPress/GlotPress-WP
# Site tour: Use driver.js

## Problem
For more flexibilities and out of the box features, we are trying out the driver.js library.


## Solution

Include the driver.js library to power the site tour.

## Testing Instructions
- Checkout this branch in your local GlotPress
- Use this sample filter
```
add_filter( 'gp_tour', function( ) {
	return array(
		'ui-intro' => [
			[
				'title' => 'UI Introduction Tour',
				'color' => '#3939c7',
			],
			[
				'selector' => '.strings .original',
				'html' => 'This is the original string that needs translation'
			],
			[
				'selector' => 'textarea.foreign-text',
				'html' => 'Enter translation here',
			],
			[
				'selector' => '.sidebar-tabs',
				'html' => 'See meta tabs here',
			],
			[
				'selector' => '.button.ok',
				'html' => 'Click here to save translation',
			]
		],
		
	);
} );
```
## Screenshots or screencast
![using-driverjs](https://github.com/GlotPress/GlotPress/assets/2834132/4d2a1ce7-0b11-4b06-a085-193e12c09932)
